app/lib/common/widgets/edit_html_description_sheet.dart

```diff
@@ -69,9 +68,10 @@ class _EditHtmlDescriptionSheetState
 
   @override
   Widget build(BuildContext context) {
-    return Container(
-      padding: const EdgeInsets.symmetric(horizontal: 20.0),
+    return Padding(
+      padding: MediaQuery.of(context).viewInsets,
```

Comment:
Need to do some changes in this file to avoid keyboard overlap over the BottomSheet issue.